app-backend/database/src/main/scala/com/azavea/rf/database/tables/Exports.scala

```diff
@@ -183,25 +183,27 @@ object Exports extends TableQuery(tag => new Exports(tag)) with LazyLogging {
 
     eo.flatMap({ exportOptions =>
 
-        val outDef = OutputDefinition(
-          crs = exportOptions.getCrs,
-          rasterSize = exportOptions.rasterSize,
-          render = Some(exportOptions.render),
-          crop = exportOptions.crop,
-          stitch = exportOptions.stitch,
-          source = exportOptions.source,
-          dropboxCredential = user.dropboxCredential
-        )
+      val outDef = OutputDefinition(
+        crs = exportOptions.getCrs,
+        rasterSize = exportOptions.rasterSize,
+        render = Some(exportOptions.render),
```

Comment:
There is already `import cats.implicits._` in scope, so you can use `exportOptions.render.some` // just a cats hint